cfgov/unprocessed/apps/owning-a-home/js/explore-rates/rate-checker.js

```diff
@@ -184,7 +183,7 @@ function updateView() {
           return;
         }
 
-        data.uniqueLabels = uniquePrimitives(data.labels.slice(0));
+        data.uniqueLabels = data.labels.slice(0);
```

Comment:
Should we want to have this in a unique primitives data structure in the future, this could use a native https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set